src/main/java/edu/fiuba/algo3/Arania.java

```diff
@@ -0,0 +1,46 @@
+package edu.fiuba.algo3;
+
+import java.util.Random;
+
+public class Arania extends Enemigo {
+
+    public Arania() {
+        this.velocidad(2) ;
+        this.dañoCausado(2);
+        this.energia (2);
+        this.creditosOtorgados(0);
+        this.coordenadas = getCoordenadasLargada();
+        this.acciones = new Vivo();
+    }
+
+    public Arania(Coordenadas coordenadas) {
+        this.velocidad(2) ;
+        this.dañoCausado(2);
+        this.energia (2);
+        this.creditosOtorgados(0);
+        this.coordenadas = coordenadas;
+        this.acciones = new Vivo();
+    }
+
+    public int cantidadCreditosOtorgados(int cantidadDeAraniasMuertas) {
+        Random random = new Random();
+        this.creditosOtorgados(random.nextInt(10) + 1);
+        return this.creditosOtorgados();
+    }
+
+    private Coordenadas getCoordenadasLargada() {
+        // Al mapa le pedira las coordenadas de la parcela de largada
+        return new Coordenadas(3,2);
+    }
+    public boolean recibirDanio(int unDanio){
+        if (estaVivo()){
+            this.energia = this.energia - unDanio;
+            return true;
+        }
+        return false;
+    }

```

Comment:
Usando estados esto quedaria asi
```java
public void recibirDanio(int danio){
    this.estado.recibirDanio(danio);
}
 ```
 Incluso si esta muerto podria lanzar excepcion

Reply:
En este caso, precisariamos modificar el atributo energia del enemigo desde adentro de estado. Esta bien hacer:
"this.estado.recibirDanio(danio, this)" ? o hay alguna otra forma de modificarlo desde adentro de estado?